tools/cldr-apps/src/main/webapp/surveytool.css

```diff
@@ -8,40 +8,14 @@
 /* Made with CSSEdit and Espresso2 */
 @charset "UTF-8";
 
-
 /* lang specific CSS */
 
-/*
-NOTE: See http://www.w3.org/International/questions/qa-css-lang
```

Comment:
These are instructions to someone maintaining this as to how to add a new webfont, such as done for Myanmar

Reply:
I'll restore the comment.